File: Factura.cs
public class Factura
{
    public string numPieza;
    public string descripcion;
    public int cantidad;
    public double precio;

    public Factura()
    {
        numPieza = "";
        descripcion = "";
        cantidad = 0;
        precio = 0.0;
    }

    public double obtenerMontoFactura()
    {
        return cantidad * precio;
    }
}

File: Factura.java
public class Factura {

    String numPieza;
    String descripcion;
    int cantidad;
    double precio;
    
    Factura(){
        numPieza = "";
        descripcion = "";
        cantidad = 0;
        precio = 0.0;
    }
    
    double obtenerMontoFactura (){
        return cantidad*precio;
    }
}